src/MLJTreeParzen.jl

```diff
@@ -198,40 +198,43 @@ end
 
 MLJTuning.default_n(tuning::MLJTreeParzenTuning, range) = 50
 
+vector(history::Nothing) = Trials.Trial[]
+vector(history::Vector) = history
+
+get_trialhist(history) =
+    map(history) do entry
+        trial_object = entry.metadata
+        measurement = entry.measurement[1]
+        # @ablaom asks "Is this deepcopy really necessary?":
```

Comment:
I'm not sure its "strictly" necessary, however, I didn't really want to reason too much about mutated variables and references and so on, so would rather to copy the incomplete object and return the completed copy